package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "serverless-offline-aws-eventbridge",
-  "version": "1.3.5",
+  "version": "1.4.0-beta.2",
   "description": "Serverless plugin to run eventbridge offline.",
   "main": "src/index.js",
   "author": "Ruben Kaiser",
   "license": "MIT",
   "scripts": {
+    "postinstall": "husky install",
```

Comment:
Hi @rubenkaiser I know you said to ignore the husky changes but this causes an error on install in my project 🚧 

```
error /Project/node_modules/serverless-offline-aws-eventbridge: Command failed.
Exit code: 127
Command: husky install
Arguments: 
```

We also use `husky` in our project but dont expect our dependencies to install git hooks :)

With `1.4.0-beta.2` installed, `sls offline` starts and receiving an event with nested subscriptions doesn't throw the  error I described in #12, but something else is happening when our `PutMessage` sdk call happens:

```
[1605840477616] ERROR	 (service-dev-function): EventBridge error
    message: {
      "ErrorCode": "code",
      "ErrorMessage": "message"
    }
    req: {
      "method": "POST",
      "headers": {
        "Content-Length": "18",
        "Content-Type": "application/json"
      }
    }
    requestId: "ckhpnyhuh0005isan8lkkc3ym"
    fn: {
      "functionVersion": "$LATEST",
      "logGroupName": "offline_logGroupName_for_service-dev-function",
      "logStreamName": "offline_logStreamName_for_service-dev-function",
      "memoryLimitInMB": "512"
    }
```

And our subscribers don't get invoked. I'm not sure if this is related to this change or to the install issue though

Reply:
Hi @nason, should have just removed the husky part. Wasn't aware that it would install the hooks in the dependencies, but sounds rather obvious when hearing it.

I've published a new version (beta 3). I've used the following setup that I thought resembles what you are trying to do.

```
functions:
  publishEvent2:
    handler: events.publish2
    events:
      - http:
          path: publish2
          method: get

  consumeEvent2:
    handler: events.consume2
    events:
      - eventBridge:
          eventBus: marketing
          pattern:
            source:
              - graph
            detail-type:
              - update
            detail:
              trigger:
                name:
                  - company
                event:
                  data:
                    new:
                      status:
                        - archived
                        - open

  publishEvent:
    handler: events.publish
    events:
      - http:
          path: publish
          method: get

  consumeEvent:
    handler: events.consume
    events:
      - eventBridge:
          eventBus: marketing
          pattern:
            source:
              - graph
            detail-type:
              - insert
              - manual
            detail:
              trigger:
                name:
                  - company

```

The handle file is also very simple but just for clarity on how I've published the messages:

```
import AWS from "aws-sdk";

AWS.config.update({
  apiVersion: "2015-10-07",
  accessKeyId: "YOURKEY",
  secretAccessKey: "YOURSECRET",
  region: "eu-west-1"
});

export const publish = async () => {
  try {
    const eventBridge = new AWS.EventBridge({
      endpoint: "http://127.0.0.1:4010",
      region: "eu-west-1",
    });
    const test = await eventBridge.putEvents({
      Entries: [
        {
          EventBusName: "marketing",
          Source: "graph",
          DetailType: "insert",
          Detail: JSON.stringify({
            trigger: {
              name: "company",
            }
          }),
        },
      ]
    }).promise();

  } catch (e) {
    console.error(e);
    return { statusCode: 400, body: "could not publish" };
  }
  return { statusCode: 200, body: "published" };
};

export const consume = async (event, context) => {
  console.log("CONSUME");
  console.log(event);
  return { statusCode: 200, body: "consume" };
};

export const publish2 = async () => {
  try {
    const eventBridge = new AWS.EventBridge({
      endpoint: "http://127.0.0.1:4010",
      region: "eu-west-1",
    });
    const test = await eventBridge.putEvents({
      Entries: [
        {
          EventBusName: "marketing",
          Source: "graph",
          DetailType: "update",
          Detail: JSON.stringify({
            trigger: {
              name: "company",
              event: {
                data: {
                  new: {
                    status: "archived"
                  }
                }
              }

            }

          })
        },
      ]
    }).promise();

  } catch (e) {
    console.error(e);
    return { statusCode: 400, body: "could not publish" };
  }
  return { statusCode: 200, body: "published" };
};

export const consume2 = async (event, context) => {
  console.log("consumed 2");
  console.log(event);
  return { statusCode: 200, body: "consumed 2" };
};
```

Is this also what you are after? Hopefully the new release works without Husky but I'm doubting that was the real issue here.

Reply:
Yes this is exactly it, sorry I didn't provide a more clear sample.

On beta 3 and am still getting the same issue -- I'm running into this catch https://github.com/rubenkaiser/serverless-offline-eventBridge/blob/1f3159c3ebb568614b6e21ccb775ea8d6482b3a6/src/index.js#L134-L150 with the error `TypeError: context.getRemainingTimeInMillis is not a function`, coming from Sentry

BUT I think this is because my handler is actually being invoked now, and the context object on line 135 there is empty, which is messing with https://www.npmjs.com/package/@sentry/serverless. This PR works, thank you so much!

Reply:
Good to hear it's working now 👍. I will merge this PR and release an updated version after testing it thoroughly in my own projects.